docs/src/pages/docs/api-reference/lifecycles/index.md

```diff
@@ -34,7 +34,7 @@ This is invoked when an attribute from the `observedAttributes` list is added, c
 removed. This could be useful to make network request for updating the data.
 
 ```javascript
-onAttributeChanged(element, ({ name, current }) => {
+let unsubscribe = onAttributeChanged(element, ({ name, current }) => {
```

Comment:
Can we use `const` instead of `let` here? Since is not going to change it make more sense a const.

Reply:
Done - no problem.